util/visualizer.py

```diff
@@ -33,14 +33,15 @@ def save_images(webpage, visuals, image_path, aspect_ratio=1.0, width=256):
     webpage.add_header(name)
     ims, txts, links = [], [], []
 
-    for label, im_data in visuals.items():
-        im = util.tensor2im(im_data)
-        image_name = '%s_%s.png' % (name, label)
-        save_path = os.path.join(image_dir, image_name)
-        util.save_image(im, save_path, aspect_ratio=aspect_ratio)
-        ims.append(image_name)
-        txts.append(label)
-        links.append(image_name)
+    for temp_visuals in visuals:
```

Comment:
temp_visuals = one visual group (one line of image?)

Reply:
yes !

Reply:
i can change the name with `visual_group` if clearer :)

Reply:
I think `visual_group` would be better!